src/Ombi/ClientApp/app/usermanagement/usermanagement-edit.component.html

```diff
@@ -1,68 +1,70 @@
 ﻿<div *ngIf="user">
-    <h3>User: {{user.userName}}</h3>
-    <button type="button" class="btn btn-primary-outline" style="float:right;" [routerLink]="['/usermanagement/']">Back</button>
+    <div class="user-details">
```

Comment:
Created another wrapper to enclose the entire edit user page since by default the ID that was being assigned to the page was a generated one from the routed component.